app/Resources/translations/messages.en_GB.xliff

```diff
@@ -502,18 +519,38 @@ An e-mail with your activation code has been sent to the e-mail address %email%.
       <trans-unit id="59ae5f416e832eeec457e8f73d11d196bcf9c6e5" resname="ss.second_factor.list.title">
         <source>ss.second_factor.list.title</source>
         <target>Token Overview</target>
-        <jms:reference-file line="4">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/SecondFactor/list.html.twig</jms:reference-file>
+        <jms:reference-file line="3">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/SecondFactor/list.html.twig</jms:reference-file>
       </trans-unit>
       <trans-unit id="64cd3b12ab8e092173e5e99069d38afdaf811b16" resname="ss.second_factor.remote_vet.text.are_you_sure">
         <source>ss.second_factor.remote_vet.text.are_you_sure</source>
         <target>Are you sure you want to confirm your identity through a third party?</target>
         <jms:reference-file line="9">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/remoteVet.html.twig</jms:reference-file>
       </trans-unit>
+      <trans-unit id="789743faa71a92af3b0c7580952c3a47e62048ac" resname="ss.second_factor.remote_vet.text.does_it_match">
+        <source>ss.second_factor.remote_vet.text.does_it_match</source>
+        <target>Is the supplied information correct?</target>
+        <jms:reference-file line="9">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/validation.html.twig</jms:reference-file>
+      </trans-unit>
       <trans-unit id="460752b1bf195dc38fb4fcce77b3603c6de3b7fd" resname="ss.second_factor.remote_vet.title">
-        <source>ss.second_factor.remote_vet.title</source>
+        <source>Remote vetting</source>
         <target>Confirm identity</target>
         <jms:reference-file line="4">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/remoteVet.html.twig</jms:reference-file>
       </trans-unit>
+      <trans-unit id="9c1a0c3ced900e4d33e110a5c2772ca68e85bf0f" resname="ss.second_factor.remote_vet_validation.title">
+        <source>ss.second_factor.remote_vet_validation.title</source>
+        <target>Validate information</target>
+        <jms:reference-file line="4">/../src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/views/RemoteVetting/validation.html.twig</jms:reference-file>
+      </trans-unit>
+      <trans-unit id="be7ea23287910e8499cd162adecc43ea8c072a20" resname="ss.second_factor.revoke.alert.remote_vetting_failed">
+        <source>ss.second_factor.revoke.alert.remote_vetting_failed</source>
+        <target state="new">Unable to validate the information</target>
```

Comment:
Please remove the `state="new"` attribute.

Reply:
Removed